filebeat/harvester/reader/limit.go

```diff
@@ -18,5 +18,9 @@ func (p *Limit) Next() (Message, error) {
 	if len(message.Content) > p.maxBytes {
 		message.Content = message.Content[:p.maxBytes]
 	}
+
+	if len(message.Raw) > p.maxBytes {
+		message.Raw = message.Raw[:p.maxBytes]
```

Comment:
I am on the fence with this. Raw message should not be truncated, because it is a form of processing. But this could protect users from injecting too long messages to their pipelines.

Reply:
I would go with truncating the raw message.

Reply:
I mean such a big message which could disrupt the output of Filebeat.